### DISCUSSION TITLE
Add "Public" or "Auth-Protected" to API Reference for Endpoints

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#5817] [Feature Requests]
It would be nice to see in the Docs what you get with/without authenticating to an endpoint like: `/server/info`

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
